airbyte-integrations/connectors/source-coingecko-coins/acceptance-test-config.yml

```diff
@@ -1,20 +1,28 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-coingecko-coins:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_coingecko_coins/spec.yaml"
+    tests:
+      - spec_path: "source_coingecko_coins/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.1.0"
```

Comment:
Definitely it, didn't manage to go through the code if SAT yet, but worth looking later on.

Reply:
@alafanechere hi! could you please take a look at this case? It looks like SAT is having troubles testing spec for backwards compatibility when there's no previous version

Reply:
@davydov-d, what happens if you remove the `backward_compatibility_tests_config` section?

Reply:
Let me reproduce it again and share the findings

Reply:
@alafanechere this is exactly the message we receive if this parameter is committed in SAT config:
```
E   hypothesis.errors.Unsatisfiable: Unable to satisfy assumptions of check_fake_previous_config_against_actual_spec
```

Reply:
UPD:

It was accidentally pushed to BETA instead of ALPHA, reverting this change from: 
https://github.com/airbytehq/airbyte/pull/18968/files#diff-3394238aa97d045886b68a0bb12a00a6cdea5680be6940e3c171d0018979ddf3R270

Reply:
I was not able to find the root cause of the problem but implemented a test skip if previous and actual spec are identical: https://github.com/airbytehq/airbyte/pull/20435